testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -1126,3 +1127,8 @@
     end
   end
 end
+
+When(/^I visit the grafana dashboards of this "([^"]*)"$/) do |host|
```

Comment:
There is no need ot this step.

What we need is a step visiting the grafana running in the monitoring server, not in the client.

And then, another step checking that the dashboard show a graph for a concrete client.